### PR TITLE
Feature/howto final draft

### PR DESCRIPTION
**Pull request by hayleyw7**
# Description drawing

- Finalize How-To wording 
- Please proof read to see if I missed any typos or steps
- Once approved & merged, I will also add this to the README

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactor feature (non-breaking change that modifies existing work)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?

- [x] local host 3000
- [ ] console.log()
- [ ] dev tools
- [ ] cypress testing

# Checklist:

- [x] My code follows the style guidelines of this project/linted
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/HowTo/HowTo.js [hunk 1]
```diff
@@ -56,7 +64,7 @@ const HowTo = (props) => {
         <article className='howto-home-container'>
 
           <p>
-            From either this how-to page or the Pokemon details page, you can return to the home page by clicking the "Back" button.
+            From either the How-To page or the Pokemon Details page, a user can return to the Home page by clicking the "x" icon.
           </p>
 
         </article>
```
Comment (Matt-Roden) [inline src/components/HowTo/HowTo.js:70]: good work thanks for polishing this up!

**Review by Matt-Roden — APPROVED** [review]
✍️

File: src/components/HowTo/HowTo.js (hunk 1 above)
Comment (joanafbrito) [reply src/components/HowTo/HowTo.js:70]: 🤗🤗

**Review by joanafbrito — APPROVED** [review]
Thank you for the refactoring

**Pull request merged**